### ISSUE TITLE
Incorrect parameters/options

### ISSUE THREAD

**Issue by komalsrathi** [deweylab/RSEM#31]
Hello,

I am trying the same command with two different versions and don't understand what's going wrong:

This is with version: RSEM v1.2.31 
```
$ rsem-prepare-reference \
-p 4 \
--gtf /mnt/isilon/cbmi/variome/reference/gencode/v23/gencode.v23.annotation.gtf \
/mnt/isilon/cbmi/variome/reference/hg38/hg38_no_alt.fa \
/mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38

rsem-extract-reference-transcripts /mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38 0 /mnt/isilon/cbmi/variome/reference/gencode/v23/gencode.v23.annotation.gtf None 0 /mnt/isilon/cbmi/variome/reference/hg38/hg38_no_alt.fa
Parsed 200000 lines
Parsed 400000 lines
Parsed 600000 lines
Parsed 800000 lines
Parsed 1000000 lines
Parsed 1200000 lines
Parsed 1400000 lines
Parsed 1600000 lines
Parsed 1800000 lines
Parsed 2000000 lines
Parsed 2200000 lines
Parsed 2400000 lines
Parsing gtf File is done!
/mnt/isilon/cbmi/variome/reference/hg38/hg38_no_alt.fa is processed!
198619 transcripts are extracted.
Extracting sequences is done!
Group File is generated!
Transcript Information File is generated!
Chromosome List File is generated!
Extracted Sequences File is generated!

rsem-preref /mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38.transcripts.fa 1 /mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38
Refs.makeRefs finished!
Refs.saveRefs finished!
/mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38.idx.fa is generated!
/mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38.n2g.idx.fa is generated!
```

This is with an older version 1.2.25 which I need to replicate [this](https://github.com/BD2KGenomics/toil-rnaseq#installation) pipeline: 

```
$ /mnt/isilon/cbmi/variome/bin/rsem/RSEM-1.2.25/rsem-prepare-reference \
-p 4 \
--gtf /mnt/isilon/cbmi/variome/reference/gencode/v23/gencode.v23.annotation.gtf \
/mnt/isilon/cbmi/variome/reference/hg38/hg38_no_alt.fa \ 
/mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38

rsem-extract-reference-transcripts /mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38 0 /mnt/isilon/cbmi/variome/reference/gencode/v23/gencode.v23.annotation.gtf 0 /mnt/isilon/cbmi/variome/reference/hg38/hg38_no_alt.fa
Usage: rsem-extract-reference-transcripts refName quiet gtfF sources hasMappingFile [mappingFile] chromosome_file_1 [chromosome_file_2 ...]
"rsem-extract-reference-transcripts /mnt/isilon/cbmi/variome/reference/rsem_hg38/hg38 0 /mnt/isilon/cbmi/variome/reference/gencode/v23/gencode.v23.annotation.gtf 0 /mnt/isilon/cbmi/variome/reference/hg38/hg38_no_alt.fa" failed! Plase check if you provide correct parameters/options for the pipeline!
```

**Comment by bli25wisc** [comment]
Hi @komalsrathi , have you compiled RSEM v1.2.25? Here is the usage information I extracted from the error report above for RSEM v1.2.25:

Usage: rsem-extract-reference-transcripts refName quiet gtfF sources hasMappingFile [mappingFile] chromosome_file_1 [chromosome_file_2 ...]

Note that it contains an additional argument "sources" between "gtfF" and "hasMappingFile". "sources" was added in later versions of RSEM but not v1.2.25. 

What I suspect is that you installed a later version globally and you might forget to compile RSEM v1.2.25 by typing 'make'. Thus, RSEM uses 'rsem-extract-reference-transcripts' from the later version.

Hope it helps,
Bo
